go/tricorder/messages/duration.go

```diff
@@ -26,6 +26,10 @@ func (d Duration) asGoDuration() time.Duration {
 	return time.Second*time.Duration(d.Seconds) + time.Duration(d.Nanoseconds)*time.Nanosecond
 }
 
+func (d Duration) AsSeconds() float64 {
+	return float64(d.Seconds) + float64(d.Nanoseconds)/1000000000.0
```

Comment:
Instead of dividing, please multiply by 1e-9. It's clearer (no counting of zeros required:-).